tests/system/runtimes/test_nuclio.py

```diff
@@ -112,6 +112,43 @@ def test_serving_with_child_function(self):
         function.deploy()
 
 
+@tests.system.base.TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestNuclioRuntimeWithKafka(tests.system.base.TestMLRunSystem):
+    project_name = "nuclio-kafka-project"
+    topic = "TestNuclioRuntimeWithKafka"
+    brokers = os.getenv("MLRUN_SYSTEM_TESTS_KAFKA_BROKERS")
+
+    @pytest.mark.skipif(
+        not brokers, reason="MLRUN_SYSTEM_TESTS_KAFKA_BROKERS not defined"
+    )
+    def test_serving_with_kafka_queue(self):
+        code_path = str(self.assets_path / "nuclio_function.py")
+        child_code_path = str(self.assets_path / "child_function.py")
+
+        self._logger.debug("Creating nuclio function")
+        function = mlrun.code_to_function(
+            name="function-with-child-kafka",
+            kind="serving",
+            project=self.project_name,
+            filename=code_path,
+        )
+
+        graph = function.set_topology("flow", engine="async")
+
+        graph.to(">>", "q1", path=f"/{self.topic}", bootstrap_servers=self.brokers).to(
```

Comment:
Is the `/` mandatory? Or can you just provide the topic name here? I think `urlparse` would still understand it as a "local path" and will work properly in that case, wouldn't it?

Reply:
Yeah, you're right, not necessary.